CMakeLists.txt

```diff
@@ -4,104 +4,200 @@
 # Support for special builds (json checking, tests, other stuff)
 
 cmake_minimum_required(VERSION 2.8)
-project(CataclysmDDA)
-set(CMAKE_MODULE_PATH
+
+PROJECT(CataclysmDDA)
+
+SET(CMAKE_MODULE_PATH
     ${CMAKE_MODULE_PATH}
     ${CMAKE_SOURCE_DIR}/cmake
 )
 
 # Build options
-option(LUA      "Lua support (required for some mods)"  OFF)
-option(TILES    "Support for graphical tilesets"        OFF)
-option(SOUND    "Support for in-game sounds & music"    OFF)
-
-# Disable internationalization on Windows and OS X as it is not supported
-# without external libraries
-if (WIN32 OR APPLE)
-    option(LOCALIZE "Support for language localizations" OFF)
-else()
-    option(LOCALIZE "Support for language localizations" ON)
-endif()
-
-# Build settings
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
-set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -g -Wall -Wextra")
-set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -Wall -Wextra -Werror -Os -DRELEASE")
+option(LUA          "Lua support (required for some mods)"      "OFF")
+option(TILES        "Support for graphical tilesets"            "OFF")
+option(SOUND        "Support for in-game sounds & music"        "OFF")
+option(DEBUG        "Enable debug. Use it for development or bug reporting purposes." "OFF")
+option(ZLEVELS      "Enable experimental z-levels"              "OFF")
+option(USE_HOME_DIR "Use user's home directory for save files." "OFF")
+option(LOCALIZE     "Support for language localizations"        "OFF")
+
+
+MESSAGE("\n * Cataclysm: Dark Days Ahead is a roguelike set in a post-apocalyptic world.")
+MESSAGE("   _________            __                   .__                                ")
+MESSAGE("   \\_   ___ \\ _____   _/  |_ _____     ____  |  |   ___.__   ______  _____      ")
+MESSAGE("   /    \\  \\/ \\__  \\  \\   __\\\\__  \\  _/ ___\\ |  |  <   |  | /  ___/ /     \\     ")
+MESSAGE("   \\     \\____ / __ \\_ |  |   / __ \\_\\  \\___ |  |__ \\___  | \\___ \\ |  Y Y  \\    ")
+MESSAGE("    \\______  /\(____  / |__|  \(____  / \\___  >|____/ / ____|/____  >|__|_|  /    ")
+MESSAGE("           \\/      \\/             \\/      \\/        \\/          \\/       \\/     ")
+MESSAGE("                               --= Dark Days Ahead =--") 
+MESSAGE("\n * http://en.cataclysmdda.com/\n")
+
+MESSAGE(STATUS "${PROJECT} build environment -- \n")
+
+MESSAGE(STATUS "Build realm is   : ${CMAKE_SYSTEM_NAME} ${CMAKE_HOST_SYSTEM_PROCESSOR}\n")
```

Comment:
no problem.

jfyi: I had to remove DSILENT in a favour of DRELEASE. In order to suppress ASCII and logging stuff set -DRELEASE=OFF

This will add ability to run resulting binary from cmake build directory as well.